pkg/destroy/gcp/bucket.go

```diff
@@ -1,12 +1,19 @@
 package gcp
 
 import (
+	"regexp"
+
 	"github.com/pkg/errors"
 
 	"google.golang.org/api/googleapi"
 	storage "google.golang.org/api/storage/v1"
 )
 
+var (
+	// multiDashes is a regexp matching multiple dashes in a sequence.
+	multiDashes = regexp.MustCompile(`-{2,}`)
+)
+
 func (o *ClusterUninstaller) listBuckets() ([]cloudResource, error) {
 	return o.listBucketsWithFilter("items(name),nextPageToken", o.ClusterID+"-", nil)
```

Comment:
I wonder if `o.ClusterID+"-"` could just be `o.ClusterID`. It might not make any difference after these changes, but just saying the extra `-` seems unnecessary...